public/js/graph/objects/node.js

```diff
@@ -152,16 +142,11 @@ Node.prototype.arePrereqsSatisfied = function () {
 
     var sat = this.checkFCEBasedPrerequisites();
     if (this.logicalType.toUpperCase() === 'AND') {
-        for (var i = 0; i < this.parents.length; i++) {
-            sat = sat && this.parents[i].isSelected();
-        }
+        sat = sat && this.parents.every(elem => elem.isSelected());        
     } else if (this.logicalType.toUpperCase() === 'OR') {
-        sat = false;
-        for (var i = 0; i < this.parents.length; i++) {
-            sat = sat || this.parents[i].isSelected();
-        }
+        sat = sat && this.parents.some(elem => elem.isSelected());
```

Comment:
I wouldn't be surprised.

You're using the new ES6 fat arrows, but I'd like to avoid ES6 until we get babel up and running (later).